trinity/utils/logging.py

```diff
@@ -42,6 +47,13 @@ def __init__(self, fmt: str, datefmt: str) -> None:
 
     def format(self, record: logging.LogRecord) -> str:
         record.shortname = record.name.split('.')[-1]  # type: ignore
+
+        if record.levelno >= logging.ERROR:
+            return bold_red(super().format(record))
+
+        if record.levelno >= logging.WARNING:
```

Comment:
*nitpick* but the return statements make these *implicit* `elif` statements.  Maybe go ahead and just make this one single `if/elif/else` block?